src/app/modules/form/admin-form/admin-form.assistance.controller.ts

```diff
@@ -0,0 +1,66 @@
+import { ChatResponseMessage } from '@azure/openai/types/openai'
+import { Joi } from 'celebrate'
+
+import { ErrorDto } from '../../../../../shared/types'
+import { ContentTypes } from '../../../../../shared/types/assistance'
+import { ControllerHandler } from '../../core/core.types'
+
+import {
+  generateFormFields,
+  generateQuestions,
+} from './admin-form.assistance.service'
+
+/**
+ * Handler for POST /questions
+ * @returns 200 when questions are successfully generated
+ * @returns 500 when openai server error occurs
+ */
+
+// Validate inputs by the user for form builder:
+const generateQuestionsSchema = Joi.object({
+  type: Joi.string()
+    .valid(...Object.values(ContentTypes))
+    .required(),
+  content: Joi.string().when('type', {
+    is: ContentTypes.PROMPT,
+    then: Joi.string().max(300).required(),
```

Comment:
ah, this is where the 300 is validated. maybe we should create a constants file in `shared/constants/assistance.ts` or something like that for this number (and can be shared with other shared constants too)